exec/cnex/global.c

```diff
@@ -127,6 +158,63 @@ void concatBytes(TExecutor *exec)
     push(exec->stack, r);
 }
 
+void nint(TExecutor *exec)
+{
+    Number a = top(exec->stack)->number; pop(exec->stack);
+
+    push(exec->stack, cell_fromNumber(number_trunc(a)));
+}
+
+void format(struct tagTExecutor *exec)
+{
+    TString *fmt   = string_fromString(top(exec->stack)->string); pop(exec->stack);
+    TString *str   = string_fromString(top(exec->stack)->string); pop(exec->stack);
+
+    Spec *spec = malloc(sizeof(Spec));
```

Comment:
Why is this done using `malloc`? Seems to be no reason for that. There's a memory leak because of this too.